website/docs/from_provider/provider_vs_riverpod.mdx

```diff
@@ -191,7 +131,7 @@ Inside the `build` method, use "watch". Inside click handlers and other events,
 use "read".
 :::
 
```

Comment:
We may want a section talking about context.select -> ref.watch(provider.select)